### ISSUE TITLE
useState lazy init

### ISSUE THREAD

**Issue by dy** [WebReflection/augmentor#19]
According to the [useState docs](https://reactjs.org/docs/hooks-reference.html#lazy-initial-state), it can be initialized with the function.

```js
let [value, setValue] = useState(() => initialValue)
// typeof value === 'function'
// but must be initialValue
```

Augmentor doesn't seem to support that for now.

[Sandbox](https://codesandbox.io/s/musing-moon-eivuz?fontsize=14&hidenavigation=1&theme=dark)

**Comment by WebReflection** [comment]
Thanks, and this is trivial too. Do you know if other hooks have the same so I can release a version that lazy initialize them all?

**Comment by WebReflection** [comment]
Never mind, I've read through and that's the only lazy value to initialize. Published as 1.1.5
